core/src/main/java/com/expedia/www/haystack/client/Tracer.java

```diff
@@ -259,10 +314,16 @@ private long calculateStartTime() {
         private Clock clock = new SystemClock();
         private Dispatcher dispatcher;
         private PropagationRegistry registry = new PropagationRegistry();
+        private Metrics metrics;
+
+        public Builder(MetricsRegistry registry, String serviceName, Dispatcher dispatcher) {
+            this(new Metrics(registry, "com.expedia.www.haystack.client.Tracer", Collections.emptyList()), serviceName, dispatcher);
```

Comment:
Tracer.class.getSImpleName() instead of the hardcoded string?

Reply:
sure.  I cribbed that in late last night to make the dw app `/metrics` output look sane... that's a much better solution. 👌